.coderabbit.yaml

```
@@ -0,0 +1,61 @@
+# Conventions de nommage
+naming
+  class PascalCase
+  method PascalCase
+  private_field _camelCase
+  local_variable camelCase
+  constant ALL_CAPS
+
+# Analyse UnityECS
+unity
+  disallow_logic_in_mono_behaviours true  # Pas de logique critique dans MonoBehaviours
+  ecs
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 0 but found 2

(indentation)